src/examples/graph.js

```diff
@@ -222,7 +222,6 @@ type IGraphProps = {};
 
 type IGraphState = {
   graph: any,
```

Comment:
If possible, pls don't use `any`, instead of mentioned relevant type. Let me know if you need any help, Thanks.